src/Tribe/Tickets.php

```diff
@@ -176,6 +176,124 @@ public function get_ticket( $event_id, $ticket_id ) {
 		}
 
 		/**
+		 * Returns the html for the delete ticket link
+		 *
+		 * @since TBD
+		 *
+		 * @param obj $ticket Ticket object
```

Comment:
Type should be `object` if we can't be more specific than that _("this is an object or any type"_ ... whereas `obj` is seen as _"an object of type obj",_ which doesn't exist)